### DISCUSSION TITLE
Can Postal be used as a mail relay?

### DISCUSSION THREAD

**Discussion by ay0ks** [postalserver/postal#3076] [Help with using Postal] [answered]
I have several mail servers in an internal network that is hosted on AWS, the main problem I am facing is that Amazon is refusing to create a PTR record for me, so those mail servers are unusable without a relay.
The title says it all, can Postal be used as a mail relay for remote mail servers to send mail through?

**Comment by willpower232** [comment, marked as answer]
You could probably make it work as a relay but it wasn't really intended for that purpose so you are likely to have a tough time figuring out any quirks that come up.

**Reply by MexHigh** [reply]
Wait, isn't Postal exactly that? I and some self-hosting friends of mine have Postal in use for some months now to solve exactly this problem with the PTR record. I'm not having a tough time, and I don't see how Postal is not intended for this use case.

Since https://docs.postalserver.io/ says "Think Sendgrid, Mailgun or Postmark [...]" (which are mail relays), I don't understand what you mean by that.

**Comment by tomcreasey** [comment]
I know I have asked this before, but having the ability to use postal as a Relay/Smart Host would be great. I want to run my own hosting servers, and I would like to use a Relay/Smart Host to send emails from the servers. I know there are solutions already, but I would like to manage the service on my servers.

I appreciate that this is different from what Postal was intended for, and for it to include this would require quite a lot of development work.

**Reply by benstaf** [reply]
@MexHigh @ay0ks The main work to do for having a Relay is to get Hard/Soft Bounces webhooks, see issue https://github.com/postalserver/postal/issues/3089

Also it would be nice to have webhooks in SendGrid format, for compatibility with Mailing list softwares:
https://github.com/orgs/postalserver/discussions/3096


You are welcome to contribute
